fs_file/fields.py

```diff
@@ -277,19 +277,17 @@ def read(self, records):
     def create(self, record_values):
         if not record_values:
             return
-        env = record_values[0][0].env
-        with env.norecompute():
-            for record, value in record_values:
-                if value:
-                    cache_value = self.convert_to_cache(value, record)
-                    attachment = self._create_attachment(record, cache_value)
-                    cache_value = self._convert_attachment_to_cache(attachment)
-                    record.env.cache.update(
-                        record,
-                        self,
-                        [cache_value],
-                        dirty=False,
-                    )
+        for record, value in record_values:
```

Comment:
Why do you remive the call to `norecompute`? It was there on purpose..

Reply:
Hi, it is deprecated: https://github.com/odoo/odoo/blob/17.0/odoo/api.py#L797